src/demo-app/input/input-demo.html

```diff
@@ -49,6 +49,49 @@
 </md-card>
 
 <md-card class="demo-card demo-basic">
+  <md-toolbar color="primary">Error messages</md-toolbar>
+  <md-card-content>
+    <h4>Regular</h4>
+
+    <p>
+      <md-input-container>
+        <input mdInput placeholder="example" [(ngModel)]="errorMessageExample1" required>
+        <md-error>This field is required</md-error>
+      </md-input-container>
+
+      <md-input-container>
+        <input mdInput placeholder="email" [formControl]="emailFormControl">
```

Comment:
the error animation is kind of strange in this case, the error message seems to appear in the middle of the input and then move down (I think because the message wraps on to 2 lines).